### DISCUSSION TITLE
Working on subset of data

### DISCUSSION THREAD

**Discussion by vijayakuruba** [p-lambda/wilds#116] [Q&A] [answered]
Hi,

Thank you for providing the WILDS benchmark. It has been really helpful, very clean and easily reproducible benchmarks.

I have a question regarding working on fraction of dataset for fast development on a small dataset.
When we take fraction of dataset, does the original classes/species and proportion of camera traps are preserved?  How do we ensure this? Looks like the code is  randomly sampling the split. Please let me know.
Thank you!

**Comment by ssagawa** [comment, marked as answer]
Thank you for your kind words and your interest! Yes, we randomly sample examples when we take a fraction of the dataset: https://github.com/p-lambda/wilds/blob/a7a452c80cad311cf0aabfd59af8348cba1b9861/wilds/datasets/wilds_dataset.py#L58. So the proportions of classes and cameras are roughly preserved, modulo random variation.

**Reply by vijayakuruba** [reply]
Thank you!
